README.md

```diff
@@ -50,6 +51,7 @@ place to ask about it might be in [ipfs/apps](https://github.com/ipfs/apps) or
 * [ipfs-share](https://github.com/rameshvarun/ipfs-share) - Pastebin/Image host/File sharing application
 * [ipfs.pics](https://github.com/ipfspics/ipfspics-server) - Upload and share pics.
 * [IPFSBin](https://github.com/victorbjelkholm/ipfsbin) - Pastebin clone build.
+* [IPFSStore](https://ipfsstore.it) - File pinning for bitcoin
```

Comment:
Yeah, you input a hash, get its associated bitcoin address, pay that and the site pins the hash for you so you know it's always online and accessible.